File: blackjack.js
import ps from "https://dev.jspm.io/prompt-sync";
const input = ps();

// Dealing hands.
let hand = (Math.floor(Math.random() * (Math.floor(22) - Math.ceil(1) + 1)) + Math.ceil(1));
let dealer = (Math.floor(Math.random() * (Math.floor(22) - Math.ceil(1) + 1)) + Math.ceil(1))

while (hand < 21 && dealer < 21) {

    console.log(`\nDelaer hand:     Your hand:\n\n${dealer}          ${hand}\n\n`)

    let move = input("Enter your move (stand/hit)\n\n>").toLowerCase()

    // Drawing.
    if (move === "hit") hand += (Math.floor(Math.random() * (Math.floor(13) - Math.ceil(1) + 1)) + Math.ceil(1))

    else if (move === "stand") dealer += (Math.floor(Math.random() * (Math.floor(13) - Math.ceil(1) + 1)) + Math.ceil(1))

    else console.log("Unknown command.")

    console.log(`\nDelaer hand:     Your hand:\n\n${dealer}          ${hand}\n\n`)

}

// Conclusion.
if (hand > 21) console.log(`\nDelaer hand:     Your hand:\n\n${dealer}          ${hand}\n\nBust. You lost!`)

else if (dealer > 21) console.log(`\nDelaer hand:     Your hand:\n\n${dealer}          ${hand}\n\nDealer bust. Victory!`)

else if (hand === dealer) console.log(`\nDelaer hand:     Your hand:\n\n${dealer}          ${hand}\n\nPush. Noone won!`)

else if (hand === 21) console.log(`\nDelaer hand:     Your hand:\n\n${dealer}          ${hand}\n\nBlackjack. Victory!`)

else console.log(`\nDelaer hand:     Your hand:\n\n${dealer}          ${hand}\n\nDealer blackjack. You lost!`)

File: blackjack.py
from random import randint

# Dealing hands.
hand = randint(1,22)
dealer = randint(1,22)

while hand < 21 and dealer < 21:

    print(f"\nDealer hand:     Your hand:\n\n{dealer}               {hand}\n\n")

    move = input("Enter your move (stand/hit)\n\n>").lower()
  
    if move == "hit":

        # Drawing.
        hand += randint(1,13)

    elif move == "stand":

        dealer += randint(1,13)

    else:

        print("Unkown command.")

    print(f"\nDelaer hand:     Your hand:\n\n{dealer}            {hand}\n\n")

# Conclusion.
if hand > 21:

    print(f"\nDelaer hand:     Your hand:\n\n{dealer}            {hand}\n\nBust! You lost.")

elif dealer > 21:

    print(f"\nDelaer hand:     Your hand:\n\n{dealer}            {hand}\n\nDealer bust! Victory.")

elif hand == dealer:

    print(f"\nDelaer hand:     Your hand:\n\n{dealer}            {hand}\n\nPush! Noone won.")

elif hand == 21:

    print(f"\nDelaer hand:     Your hand:\n\n{dealer}            {hand}\n\nBlackjack! Victory.")

else:

    print(f"\nDelaer hand:     Your hand:\n\n{dealer}            {hand}\n\nDealer blackjack! You lost.")